index.js

```diff
@@ -28,7 +44,7 @@ function Agent(callback, _opts) {
   // (i.e. it has a callback function) lazily
   this._promisifiedCallback = false;
 
-  let opts = _opts;
+  var opts = _opts;
```

Comment:
Why `var`?

Reply:
let inside blocks breaks some of the environments that are supposed to work on CI (here and in this other PR: https://github.com/TooTallNate/node-https-proxy-agent/pull/71 ). I assumed that making all vars would just help us keep consistency! I can change it back for sure, just let me know.